### ISSUE TITLE
CSS Animation and OBS

### ISSUE THREAD

**Issue by Shi974** [Nerixyz/current-song-overlay#13]
Hey there ! First of all, I want to say a huge thank you for this little plugin ! It is amazing ! 🤩
I just have one issue with it (probably an issue with OBS and not the plugin, but maybe I'm not the only one ?)
The CSS animation work correctly in my browser but it seems like it's not working with OBS and so the text is truncated.

I played a little with the CSS and the JS but it seems it's still working in my browser, so, am I the only one with an OBS that does not show CSS keyframes ?

<sub><sup>*It's a minor thing, not even a real issue but I thought it couldn't hurt to ask*</sup></sub>

Thanks again for this ❤

**Comment by Nerixyz** [comment]
🤔

To clarify:
1. Is the text not moving?
2. Is the text moving but not the entire text is shown throughout the animation?

Another quesion: Is the text "faded" out on the sides?

If you could provide a screenshot, then it would be nice.

Also, you could try to play with the witdth and height of the overlay and the `--song-info-max-width` CSS variable.

You can try to use chrome debugger (See [this post](https://obsproject.com/forum/threads/how-to-get-console-log-in-browsersource.110221/)). This way you can inspect the chromium instance in the browser.

**Comment by Shi974** [comment]
Thanks for your quick answer!

The text is not moving (in OBS). // [Image](https://ibb.co/7WZYMsB)
(When I open the HTML file in my browser, the moving text animation is just fine.)

So I am wondering if OBS needs any kind of webkit properties ?

This is what I did in my modified CSS file in the meantime. :) I removed the max-width and it shows entirely, but I like the blocked width and animated version.
[Image1](https://ibb.co/ZYhWskL) // [Image2](https://ibb.co/mS8HmHt )

But, just like I said, the plugin is working correctly and this "issue" is more about OBS and CSS keyframes compability.
Is the animation moving correctly in your OBS ?

Also, this is another topic (sorry) : is there a possibility to block a custom audio source from the browser ? For instance, my streambot sounds are caught by the plugin and are shown during stream, even if a song is playing (and shown by the plugin).
